04 - Guides, Workflows, & Courses/Guides/How to get the most out of the Breadcrumbs plugin.md

```diff
@@ -0,0 +1,78 @@
+
+#### Setting your hierarchy note
+
+I started using the plugin when the first Alternative Hierarchy feature launched: the [Hierarchy Note](https://github.com/SkepticMystic/breadcrumbs/wiki/Alternative-Hierarchies#hierarchy-notes). This appears in the settings under Hierarchy Notes. In my opinion it's not sustainable to exclusively use the hierarchy note as the list gets way too long, and you have to manually add things, but I think it's a good starting point, especially if you're still just testing the plugin.
+
+#### Figuring out your hierarchies
+
+Start with what you already have. Even if you subscribe to a flat-structure vault, your notes likely already have some sort of hierarchy. When you notice an existing hierarchy, consider adding it to the list of Hierarchies. If you follow what feels most intuitive to you, you're more likely to stick to it. It shouldn't be a chore and you probably don't need to restructure your entire vault to accommodate Breadcrumbs.
+
+If you use Breadcrumbs, you likely already use the [Dataview plugin](https://github.com/blacksmithgu/obsidian-dataview/). As of v. 1.9.0, you can use [Folder indexes](https://github.com/SkepticMystic/breadcrumbs/wiki/Alternative-Hierarchies#folder-indexes) and, as of v. 1.10.0, you can use [Tag indexes](https://github.com/SkepticMystic/breadcrumbs/wiki/Alternative-Hierarchies#tag-indexes). So you can build Breadcrumbs hierarchies from folders, notes or dataview fields – things you probably already have in some form in your vault.
+
+(A disclaimer: when I discovered Breadcrumbs I did spend an entire afternoon, and maybe the evening and the next day too, playing with the plugin and manually adding hierarchical fields to each of my hundreds of notes. Don't be like me. Please value your time.)
+
+#### Folders > Tags > YAML > Inline
+
+If you're concerned about portability (I know I am), take this nifty and entirely made up hierarchy from most portable way to create breadcrumbs to least portable: Folders > Tags > YAML > Inline. 
+
+Aside from my most top-level notes (the Homepage, the note for >> LAW <<, etc), I almost exclusively use tags and inline links. Tags are intuitive to me and very compact, and inline links are non-obstructive to the general content of the note as well as being explicit, unlike YAML. 
+
+I personally find folders constraining but I think the big pushback has gone a bit too far: they absolutely have their uses, and the "folders are bad" narrative has made some people avoid folders when they would clearly benefit from them. Folders are also the most portable way to see your hierarchies and your files. Just check your system file explorer.
+
+#### Hierarchy ideas
+
+For some ideas on different hierarchies, these are the ones I currently use (some could do with better names, I know):
```

Comment:
I think it would help to have a screenshot showing one or two of these in action, for those who are unfamiliar - or not very familiar with - the Breadcrumbs plugin.

Sadly, due to my own lack of knowledge of Breadcrumbs, personally, I don't understand the purpose or benefits of any of these...

An alternative to this, which may be worthwhile anyway, is to put a note at the top of the page conveying who the target audience is...

For example:

> Target audience: people who have already used the Breadcrumbs and understand its concepts of ... [list of concepts mentioned in this article] and how to use them, and would like ideas for how to use them effectively.

Reply:
I'm also going to mark this one as solved. Maybe @lkadre can add a screenshot or two later on in a separate PR!